### PR TITLE
Update vocdoni/sdk 0.0.15

### PR DESCRIPTION
**Pull request by antopalidi**
The new version of Vocdoni 0.0.15 adds a new function `calculateElectionCost`:

```
  public calculateElectionCost(election: UnpublishedElection): Promise<number> {
    return this.fetchChainId()
      .then(() =>
        ElectionAPI.price(
          this.url,
          election.maxCensusSize,
          ElectionCore.estimateElectionBlocks(election, this.chainData),
          election.electionType.secretUntilTheEnd,
          election.electionType.anonymous,
          election.voteType.maxVoteOverwrites
        )
      )
      .then((cost) => cost.price);
  }
```
It was necessary to change the value of the constant `MAX_VOTE_OVERWRITES` from -1 to 0 to calculate the cost correctly

### CODE REVIEW

**Comment by antopalidi** [comment]
@andreslucena this pr is ready for review

File: app/packs/src/decidim/vocdoni/admin/utils/create_vocdoni_election.js [hunk 1]
```diff
@@ -2,7 +2,7 @@ import { Election, PlainCensus } from "@vocdoni/sdk";
 import { initVocdoniClient } from "src/decidim/vocdoni/admin/utils/init_vocdoni_client";
 
 // Allow the user to vote multiple times
```
Comment (andreslucena) [inline app/packs/src/decidim/vocdoni/admin/utils/create_vocdoni_election.js:4]: As we're changing the value for vote overwrite, we need to update accordingly the comment:

```suggestion
// Do not allow the user to vote multiple times
```

**Review by andreslucena — APPROVED** [review]
Now it leaves me setup the Election, and also I've seen that you've fixed the nokogiri problem that we had in the CI, so great 👏🏽 👏🏽 

The change in the `MAX_VOTE_OVERWRITES` introduces a bug where we're saying that you can overwrite the vote when you actually can't do that: 


https://github.com/decidim-vocdoni/decidim-module-vocdoni/assets/717367/16caa810-ea90-49f2-9670-019f6e0fa468

But as this is a story that we have in the backlog we can tackle this problem when we arrive to that issue (it isn't refined and created yet).

**Review by andreslucena — CHANGES_REQUESTED** [review]
Oh I've approved it but I actually wanted to "request changes" to fix the comment in the code. Sorry!

**Comment by antopalidi** [comment]
@andreslucena 

Yes, I know this will be implemented and improved later. So for now I suggested to set `MAX_VOTE_OVERWRITES = 0` to avoid test errors when creating elections, because this parameter cannot be -1

Changes added

File: app/packs/src/decidim/vocdoni/admin/available_credits.js [hunk 2]
```diff
@@ -46,6 +46,12 @@ const collectFaucetTokensListener = async () => {
   });
 }
 
+export const getAvailableCredits = async () => {
```
Comment (andreslucena) [inline app/packs/src/decidim/vocdoni/admin/available_credits.js:49]: I think we should refactor the showAvailableCredits function, for remove the duplication of logic. I didn't actually tried but this should be enough:  

```javascript
const showAvailableCredits = async () => {
  const creditsSpan = document.querySelector(CREDIT_SPAN_SELECTOR);
  if (!creditsSpan) {
    return;
  }

  const availableCredits = await getAvailableCredits();

  if (availableCredits === 0) {
    document.querySelector(NO_TOKENS_MESSAGE_SELECTOR).classList.remove("hide");
    document.querySelectorAll(ACTIONS_BUTTONS_SELECTOR).forEach((element) => {
      element.disabled = true;
    });
  }

  creditsSpan.innerHTML = availableCredits;
};
```

**Review by andreslucena — CHANGES_REQUESTED** [review]
I have a refactor suggestion with the credits change, that will remove a bit of duplication. Can you check it out?

File: app/packs/src/decidim/vocdoni/admin/available_credits.js (hunk 2 above)
Comment (antopalidi) [reply app/packs/src/decidim/vocdoni/admin/available_credits.js]: I refactored it.

**Review by andreslucena — APPROVED** [review]
LGTM

**Pull request merged**